ios/reactNativeApp/Info.plist

```diff
@@ -1,81 +1,81 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
+<dict>
```

Comment:
I didn't do anything with this file, seems like Xcode insisted on reformatting it 🤷‍♂️